CEDS/MSSQL/0037-View-FactK12ProgramParticipationsDim-Create.sql

```diff
@@ -0,0 +1,157 @@
+            analytics.ceds_K12SchoolDim
+        ON
+            ceds_K12SchoolDim.K12SchoolKey = StudentSpecialEducationProgramAssociation.EducationOrganizationId
+        INNER JOIN
+            edfi.ProgramTypeDescriptor
+        ON
+            ProgramTypeDescriptor.ProgramTypeDescriptorId = StudentSpecialEducationProgramAssociation.ProgramTypeDescriptorId
+        INNER JOIN
+            edfi.Descriptor
+        ON
+            Descriptor.DescriptorId = ProgramTypeDescriptor.ProgramTypeDescriptorId
+        INNER JOIN
+            analytics.ceds_K12ProgramTypeDim
+        ON
+            Descriptor.CodeValue = ceds_K12ProgramTypeDim.ProgramTypeCode
+        INNER JOIN
+            edfi.Student
+        ON
+            Student.StudentUSI = StudentSpecialEducationProgramAssociation.StudentUSI
+        INNER JOIN
+            analytics.ceds_K12StudentDim
+        ON
+            ceds_K12StudentDim.StudentIdentifierState = Student.StudentUniqueId
+        LEFT JOIN
+            edfi.StudentEducationOrganizationAssociation
+        ON
+            StudentEducationOrganizationAssociation.StudentUSI = StudentSpecialEducationProgramAssociation.StudentUSI AND Descriptor.Namespace = StudentEducationOrganizationAssociation.SexDescriptorId
+        INNER JOIN
+            edfi.StudentEducationOrganizationAssociationStudentCharacteristic
+        ON
+            StudentEducationOrganizationAssociationStudentCharacteristic.EducationOrganizationId = StudentSpecialEducationProgramAssociation.EducationOrganizationId
+        INNER JOIN
+            edfi.StudentHomelessProgramAssociation
+        ON
+            StudentHomelessProgramAssociation.StudentUSI = StudentEducationOrganizationAssociation.StudentUSI
+        INNER JOIN
+            analytics.ceds_K12DemographicDim
+        ON
+            ceds_K12DemographicDim.K12DemographicKey = StudentHomelessProgramAssociation.ProgramName
+        INNER JOIN
+            edfi.GeneralStudentProgramAssociation
+        ON
+            GeneralStudentProgramAssociation.BeginDate = StudentSpecialEducationProgramAssociation.BeginDate OR
+            GeneralStudentProgramAssociation.EducationOrganizationId = StudentSpecialEducationProgramAssociation.EducationOrganizationId OR
+            GeneralStudentProgramAssociation.ProgramEducationOrganizationId = StudentSpecialEducationProgramAssociation.ProgramEducationOrganizationId OR
+            GeneralStudentProgramAssociation.ProgramName = StudentSpecialEducationProgramAssociation.ProgramName OR
+            GeneralStudentProgramAssociation.ProgramTypeDescriptorId = StudentSpecialEducationProgramAssociation.ProgramTypeDescriptorId OR
+            GeneralStudentProgramAssociation.StudentUSI = StudentSpecialEducationProgramAssociation.StudentUSI
+        INNER JOIN
+            analytics.ceds_IdeaStatusDim
+        ON
+            ceds_IdeaStatusDim.DisabilityCode = Descriptor.CodeValue AND ceds_IdeaStatusDim.BasisOfExitCode = Descriptor.CodeValue
+        )
```

Comment:
I think there might be a join pending. The mapping document mentions a join to `edfi.StudentSpecialEducationProgramAssociationDisability` that I don't see here.